R/test-files.r

```diff
@@ -41,12 +41,16 @@ test_files <- function(paths, reporter = "summary",
 
   current_reporter <- find_reporter(reporter)
   current_reporter$start_reporter()
+
   results <- lapply(paths, test_file, env = env,
     reporter = current_reporter, start_end_reporter = FALSE)
+
   current_reporter$end_reporter()
 
-  results <- unlist(results, recursive = FALSE)
+  # TODO: Integrate results.
+  compiled_results <- test_compiled_code(reporter = reporter)
```

Comment:
Because the tests are a property of the package itself (and so perhaps this isn't really the most natural place to fit it, but I couldn't think of a better place)